packages/@uppy/server-utils/src/Provider.js

```diff
@@ -52,4 +52,27 @@ module.exports = class Provider extends RequestClient {
         return res
       })
   }
+
+  static initPlugin (plugin, opts, defaultOpts) {
```

Comment:
This static function serves as a place where setups common to all Provider Plugins is done. Please let me know if you have suggestions for a better place where this function can live.

PS: I don't know of a different way to approach this. It's either we do it the way I'm doing it here. Or we have a class called `ProviderPlugin` which all Provider Plugins extend. Otherwise, we would continue to have repetitive code in all providers, which is hard to maintain.

Reply:
This seems like a pretty good way to do it—personally I would keep the `type = 'acquirer'` and the default options merging duplicated in each Plugin constructor for consistency with every other plugin.